commands/get.js

```diff
@@ -115,13 +159,14 @@ async function downloadResource (db, options, resource, directory, collection, r
 /**
  * download a collection from an existdb instance
  * @param {NodeExist} db NodeExist client
+ * @param {GetOptions} options
  * @param {boolean} verbose
  * @param {String} collection
  * @param {String} baseCollection
  */
 async function downloadCollection (db, options, collection, baseCollection, directory) {
   const absCollection = posix.join(baseCollection, collection)
-  const { verbose } = options
+  const { verbose, maxInflightRequests = 10 } = options
```

Comment:
The default value for parallel requests is already set further up when parsing command line options. It can be removed here